lib/onfido/configuration.rb

```diff
@@ -36,12 +36,10 @@ def logger
     end
 
     def endpoint
-      region_host = region ? REGION_HOSTS[region.downcase.to_sym] : "api.onfido.com"
-      unless region_host
-        raise "The region \"#{region.downcase}\" is not currently supported"
-      end
+      region_host = region ? REGION_HOSTS[region.downcase.to_sym] : 'api.onfido.com'
```

Comment:
As this defaults to v3.1 and `api.onfido.com` doesn't fully support that version yet, will this cause issues as our fallback?

Reply:
Fixed in my MR (which merges into this one)